k9mail/src/test/java/com/fsck/k9/mailstore/StoreSchemaDefinitionTest.java

```diff
@@ -295,11 +297,12 @@ private void assertDatabaseIndexesEquals(SQLiteDatabase expected, SQLiteDatabase
 
     private List<String> objectsInDatabase(SQLiteDatabase db, String type) {
         List<String> tables = new ArrayList<>();
-        Cursor cursor = db.rawQuery("SELECT name FROM sqlite_master WHERE type = ?", new String[] { type });
+        Cursor cursor = db.rawQuery("SELECT sql FROM sqlite_master WHERE type = ? AND sql IS NOT NULL", new String[] { type });
```

Comment:
Good point. I've found this for the index `sqlite_autoindex_messages_fulltext_segdir_1` which is automatically created internally by SQLite when creating the FTS4 table `messages_fulltext`. Therefore, we can safely ignore such a type of index.